actions/emoji_actions.js

```diff
@@ -79,16 +80,27 @@ export function addRecentEmoji(alias) {
         }
 
         let updatedRecentEmojis;
-        if (recentEmojis.includes(name)) {
-            // If the emoji is already in the recent list, remove it and add it to the front
-            const recentEmojisFiltered = recentEmojis.filter((recentEmoji) => recentEmoji !== name);
-            updatedRecentEmojis = [...recentEmojisFiltered, name].slice(-MAXIMUM_RECENT_EMOJI);
+        const currentEmojiInRecentList = recentEmojis.find((recentEmoji) => recentEmoji.name === name);
+        if (currentEmojiInRecentList) {
+            // If the emoji is already in the recent list, remove it and add it to the front with updated usage count
+            const recentEmojisFiltered = recentEmojis.filter((recentEmoji) => recentEmoji.name !== name);
```

Comment:
(0/5, may be premature optimization) If we already looked for the currentEmoji in the list, cannot we get the index, and just take that index out of the list with splice? That would make so we only traverse the recentEmojis list once.